frontend/src/App/App.jsx

```diff
@@ -44,10 +45,10 @@ export default function App() {
             <Route path="/" element={<Landing />} />
             {/* <Route path="/" element={user ? <Landing /> : <Login />} /> N/B: To confirm that User is still logged in after page refresh */}
             <Route path="/login" element={<Login />} />
-            <Route path="/myprofile" element={<MyProfile />} />
-            <Route path="/patients" element={<Patients />} />
-            <Route path="/discover" element={<Discover />} />
             <Route path="/signup" element={<Signup />} />
+            <Route path="/discover" element={<Discover />} />
+            <Route path="/myprofile" element={user ? <MyProfile />: <Landing />} />
```

Comment:
i think this is okay to keep as a **temporary** solution for now, but let's definitely circle back soon to come up with a more robust solution